packages/fxa-settings/src/pages/Signup/index.tsx

```diff
@@ -0,0 +1,490 @@
+            className="link-blue text-start text-sm mb-4"
+          >
+            Why do we ask?
+          </LinkExternal>
+        </FtlMsg>
+
+        {isCWTSEnabled && (
+          <>
+            <FtlMsg id="signup-choose-what-to-sync-heading">
+              <h2 className="text-base">Choose what to sync</h2>
+            </FtlMsg>
+            <div className="flex flex-wrap text-start ltr:mobileLandscape:ml-6 rtl:mobileLandscape:mr-6 mb-1 text-sm">
+              {engines.map((engine) => (
+                <div
+                  key={engine.id}
+                  className="mb-4 relative flex-50% rtl:mobileLandscape:pr-6 ltr:mobileLandscape:pl-6 rtl:pr-3 ltr:pl-3 flex items-center"
+                >
+                  <Checkbox
+                    label={engine.text}
+                    prefixDataTestId={engine.id}
+                    defaultChecked={engine.checked}
+                  />
+                </div>
+              ))}
+            </div>
+          </>
+        )}
+        {/* TODO: When are newsletters enabled? */}
+        {areNewslettersEnabled && (
+          <section className="flex flex-col text-start text-sm gap-4 my-4">
+            <FtlMsg id="signup-newsletter-subscription-heading">
+              <p>
+                Practical knowledge is coming to your inbox. Sign up for more:
+              </p>
+            </FtlMsg>
+            {newsletters.map((newsletter) => (
+              <Checkbox key={newsletter.slug} label={newsletter.label} />
+            ))}
+          </section>
+        )}
+      </FormPasswordWithBalloons>
+
+      {isPocketClient ? (
+        <FtlMsg id="signup-pocket-terms-privacy-agreement">
+          <p className="text-xs text-grey-500 mt-4">
+            By proceeding, you agree to:
+            <br />
+            Pocket’s{' '}
+            <LinkExternal
+              data-testid="pocket-tos-external-link"
+              href="https://getpocket.com/tos/"
+              className="link-grey"
+            >
+              Terms of Service
+            </LinkExternal>{' '}
+            and{' '}
+            <LinkExternal
+              data-testid="pocket-privacy-notice-external-link"
+              className="link-grey"
+              href="https://getpocket.com/privacy/"
+            >
+              Privacy Notice
+            </LinkExternal>
+            <br />
+            Firefox’s{' '}
+            <a
+              data-testid="fxa-tos-link"
+              href="/legal/terms"
+              className="link-grey"
+            >
+              Terms of Service
+            </a>{' '}
+            and{' '}
+            <a
+              data-testid="fxa-privacy-link"
+              href="/legal/privacy"
+              className="link-grey"
+            >
+              Privacy Notice
+            </a>
+          </p>
+        </FtlMsg>
+      ) : (
+        <FtlMsg id="signup-default-terms-privacy-agreement">
```

Comment:
Assuming Mill's PR gets in first, we should follow [the pattern](https://github.com/mozilla/fxa/pull/14769#discussion_r1086830014) set there, and likely create components out of this as you'd pointed out in a previous comment. 😀

Reply:
I'm working on pulling out Mill's updated pattern into its own component for re-use. 👍